apps/tsilhqotin-language-hub/src/Components/Pages/About/About.tsx

```diff
@@ -11,38 +12,46 @@ export function About(props: AboutProps) {
                     <h1 id="pageTitle">Dialect</h1>
                 </div>
             </div>
-            <div className="pageContent">
-                <h1>Tŝilhqot’in Ch’ih Yaltɨg</h1>
-                <div>Xenchuh ʔElhghaʔeyuwh Jid Gwetowh Gudzɨsh, ʔEguh Chuh Seʔagunt’ih</div>
-                <div>Loose translation of Tŝilhqot’in Language- Respecting our Diversity</div>
-                <audio
-                    src="https://api.tsilhqotinlanguage.ca/uploads/1719_Bella_Alphonse_Respecting_Dialect_2_4d710497b9.mp3"
-                    controls
-                    className="dialectAudio"
-                />
-                <div>
-                    Translation by Bella Alphonse with assistance from Aaron Plahn.
-                    <br /> Nexwesiqi Tŝilhqot’in ch’ih yajelhtɨg ʔeguh najedetat’i. ʔEsdilagh
-                    gwet’in, Tl’esqox-t’in, Tl’etinqox-t’in, Tŝi Deldel gwet’in, Xeni gwet’in, belh
-                    Yuneŝit’in ʔeyen Tŝilhqot’in ch’ih yajelhtɨg hajint’ih. Xenchuh Tŝilhqot’in
-                    ʔelhghaʔeyuwh jid gwetowh jeguzɨsh, ʔeguh chuh seʔagunt’ih. Naʔet’sen jid
-                    gwetowh ts’eguzih hink’ed, lha gwa huyenilẑen chu, hink’an lha deni
-                    ghanteẑindlux hanh. <br /> Lha ʔinlhanx dzanh su ʔegun jid yalhtɨg hagunt’ih.
-                    ʔElhghaʔeyuwh jid gwetowh ts’eguzih. T’agultinqi yajelhtɨg, sutsel ʔuẑilhtŝ’an;
-                    ʔeguh jid ʔigwedilʔanx hagwet’insh.
+            <Card className="pageContent">
+                <h2>Tŝilhqot’in Ch’ih Yaltɨg</h2>
+                <Divider />
+                <div style={{ padding: '20px' }}>
+                    <div style={{ paddingTop: '20px' }}>
+                        Xenchuh ʔElhghaʔeyuwh Jid Gwetowh Gudzɨsh, ʔEguh Chuh Seʔagunt’ih
```

Comment:
For now we can hard-wire this. But in the future we should find some way to pull it from the database.